ansible/roles/dev-desktop/tasks/main.yml

```
@@ -186,6 +186,11 @@
       - gcc-multilib # for 32 bit cross compiles of libstd and tests
     state: present
 
+- name: Install Nodejs
+  shell: |
+    curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+    apt-get install -y nodejs
```

Let's add a quick check to see if Node is already installed and use the `apt` module to install the package:

```suggestion
- name: Check if Node is installed
  command: node --version
  register: node_version
  changed_when: false
  failed_when: node_version.rc != 0 and node_version.rc != 2

- name: Set up NodeSource repository
  shell: |
    curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
  when: node_version.rc == 2

- name: Install Node
  apt:
    name: nodejs
    state: present
    update_cache: yes
```